todolist/src/reducers/todoReducer.js

```diff
@@ -1,23 +1,45 @@
-import { ADD_TODO, TOGGLE_TODO } from '../actions/todoActions';
+import { ADD_TODO, DELETE_TODO, TOGGLE_TODO } from '../actions/todoActions';
 
 const initialState = {
-  todos: []
+  todos: getTodosFromLocalStorage(),
 };
 
+function getTodosFromLocalStorage() {
+  const todos = localStorage.getItem('todos');
```

Comment:
use redux persist to handle local storage

Reply:
Done.